sensing/preprocessor/pointcloud/pointcloud_preprocessor/CMakeLists.txt

```diff
@@ -15,8 +17,6 @@ ament_auto_find_build_dependencies()
 find_package(OpenCV REQUIRED)
 find_package(Eigen3 REQUIRED)
 find_package(Boost REQUIRED)
-find_package(PCL REQUIRED)
```

Comment:
@jilaada was removing that line enough to fix #129 ?

Reply:
Yeah I checked the error message: https://cmake.org/cmake/help/git-stage/policy/CMP0074.html and it seemed to relate to the find_package and the use of ament_auto to find dependencies. Removing these 2 lines did the trick. I also noticed that the libraries might not be linked correctly, so I added those down below. I'm not a CMake expert so any critique is more than welcomed!

Reply:
One of the quirks of `ament_auto_find_build_dependencies()` is that it doesn't add the `REQUIRED` keyword, so if you want to require dependencies, you have switch the order. That might have caused the problem, so try

```cmake
find_package(Boost REQUIRED)
find_package(PCL REQUIRED)
ament_auto_find_build_dependencies()
```

Reply:
So I just tried this and I ended up with stderr outputs:
```
jilada@ade:~/AutowareArchitectureProposal (ros2 $% u=)$ colcon build --packages-up-to pointcloud_preprocessor --cmake-args -DCMAKE_EXPORT_COMPILE_COMMANDS=1
Starting >>> pcl_conversions
Finished <<< pcl_conversions [0.21s]
Starting >>> pcl_ros
Finished <<< pcl_ros [0.18s]
Starting >>> pointcloud_preprocessor
[Processing: pointcloud_preprocessor]
--- stderr: pointcloud_preprocessor
** WARNING ** io features related to openni will be disabled
** WARNING ** io features related to openni2 will be disabled
** WARNING ** io features related to pcap will be disabled
** WARNING ** io features related to png will be disabled
** WARNING ** io features related to libusb-1.0 will be disabled
** WARNING ** visualization features related to openni will be disabled
** WARNING ** visualization features related to openni2 will be disabled
** WARNING ** apps features related to openni will be disabled
---
Finished <<< pointcloud_preprocessor [51.6s]

Summary: 3 packages finished [52.5s]
  1 package had stderr output: pointcloud_preprocessor
```

Reply:
Yep, many packages have that. I didn't find out how to suppress them yet.